fizzbuzz.js

```diff
@@ -55,13 +58,14 @@ const fizzbuzzAlgoritmus2 = () => {
 
 /* 
 -- Brief --
-List with numbers (ul/li)
-Slyle of Fizz, Buzz and FizzBuzz at the list
-Optional number of listings
+Create a list with numbers (ul/li)
+Slyle Fizz, Buzz and FizzBuzz at the list
```

Comment:
fix errors
```suggestion
Style Fizz, Buzz and FizzBuzz in the list
```
or even better
```suggestion
Make Fizz, Buzz and FizzBuzz look differently in the list.
```